PMM-8963 Extended doc for paths base flag in pmm agent. (#608)
percona/pmm-doc: docs/details/commands/pmm-agent.md

| `-h`, `--help`                         |                                     | Show help (synonym for `pmm-agent help`).
| `--version`                            |                                     | Show application version, PMM version, time-stamp, git commit hash and branch.

## USAGE AND EXAMPLES OF `paths-base` FLAG

Since 2.23.0 you can use this flag for easier setup of pmm agent. With this flag you don't need root permissions for PMM client anymore and it will be
fully working.

**Examples:**

> - **Case 1:** I don't have root permissions to `/usr/local/percona/pmm2` folder or I want to change default folder for PMM files.

Reviewer: ```suggestion
- **Case 1:** There are no root permissions for `/usr/local/percona/pmm2` folder or there is a need to change default folder for PMM files.
```

````
Config output:
````
# Updated by `pmm-agent setup`.
---
id: /agent_id/fc546c21-0f8c-4621-b031-e228593a1d0a
listen-address: 127.0.0.1
listen-port: 7777
server:
    address: 127.0.0.1:443
    username: admin
    password: admin
    insecure-tls: true
paths:
    paths_base: /home/user/custom/pmm2
    exporters_base: /home/user/pmm2/exporters
    node_exporter: /home/user/pmm2/exporters/node_exporter
    mysqld_exporter: /home/user/pmm2/exporters/mysqld_exporter
    mongodb_exporter: /home/user/pmm2/exporters/mongodb_exporter
    postgres_exporter: /home/user/pmm2/exporters/postgres_exporter
    proxysql_exporter: /home/user/pmm2/exporters/proxysql_exporter
    rds_exporter: /home/user/pmm2/exporters/rds_exporter
    azure_exporter: /home/user/pmm2/exporters/azure_exporter
    vmagent: /home/user/pmm2/exporters/vmagent
    tempdir: /tmp
    pt_summary: /home/user/pmm2/tools/pt-summary
    pt_pg_summary: /home/user/pmm2/tools/pt-pg-summary
    pt_mysql_summary: /home/user/pmm2/tools/pt-mysql-summary
    pt_mongodb_summary: /home/user/pmm2/tools/pt-mongodb-summary
ports:
    min: 42000
    max: 51999
debug: false
trace: false
````

> As you can see above, base for all exporters and tools was changed only by setting --paths-base. With this tag you can specify folder for PMM where you don't have root access.

Reviewer: ```suggestion
As could be seen above, base for all exporters and tools was changed only by setting `--paths-base`. With this tag the folder for PMM that doesn't require root access could be specified.
```

> - **Case 2:** You will pass older --paths-exporters_base flag and also --paths-base

Reviewer: ```suggestion
- **Case 2:** The older `--paths-exporters_base` flag could be passed along with the `--paths-base`
```

````
Config output:
````
# Updated by `pmm-agent setup`.
---
id: /agent_id/afce1917-8836-4857-b3e5-ad372c2ddbe5
listen-address: 127.0.0.1
listen-port: 7777
server:
    address: 127.0.0.1:443
    username: admin
    password: admin
    insecure-tls: true
paths:
    paths_base: /home/user/custom/pmm2
    exporters_base: /home/user/exporters
    node_exporter: /home/user/exporters/node_exporter
    mysqld_exporter: /home/user/exporters/mysqld_exporter
    mongodb_exporter: /home/user/exporters/mongodb_exporter
    postgres_exporter: /home/user/exporters/postgres_exporter
    proxysql_exporter: /home/user/exporters/proxysql_exporter
    rds_exporter: /home/user/exporters/rds_exporter
    azure_exporter: /home/user/exporters/azure_exporter
    vmagent: /home/user/exporters/vmagent
    tempdir: /tmp
    pt_summary: /home/user/custom/pmm2/tools/pt-summary
    pt_pg_summary: /home/user/custom/pmm2/tools/pt-pg-summary
    pt_mysql_summary: /home/user/custom/pmm2/tools/pt-mysql-summary
    pt_mongodb_summary: /home/user/custom/pmm2/tools/pt-mongodb-summary
ports:
    min: 42000
    max: 51999
debug: false
trace: false
````

> As you can see above behavior for --paths-base was the same, but paths for all exporters were overwritten by --paths-exporter_base flag. 

Reviewer: ```suggestion
As could be seen above the behavior for the `--paths-base` was the same, but paths for all exporters were overwritten by the `--paths-exporter_base` flag. 
```

**Summary:**

> Flag --paths-base will set path for all exporters and tools, but each one can be overrided by specific flag (like --paths-mongodb_exporter, --paths-pt-mysql-summary etc).

Reviewer: ```suggestion
Flag `--paths-base` will set path for all exporters and tools, but each one could be overridden by specific flag (like `--paths-mongodb_exporter`, `--paths-pt-mysql-summary` and etc).
```
